modules/special-rules/pages/war-engines-and-blast-markers.adoc

```diff
@@ -1,11 +1,29 @@
 = War engines and Blast markers
 
-Blast markers affect War Engine detachments in much the same way as other units. 
+Blast markers affect War Engine detachments in much the same way as other units.
 However, *all* War Engine detachments have the {stubborn} ability.
 
-Note that for a War Engine detachment has a total initial Damage Capacity value of more than `15`, you may add +1 to the result of the Leadership test.
+You may add +1 to the result of Leadership tests when the total initial Damage Capacity value of the War Engine detachment is `15` or more, instead when it has `15` or more units.
+
 {blank}[TODO: Is this the best place to highlight this point? I think it might be better to improve the Leadership test text and to clarify what is what when it comes to 'number of units' and 'detachment strength' and initial Damage Capacity versus current Damage Capacity for the purposes of the Leadership test.]
 
+.Ork Great Gargant
+====
+A Great Gargant is a war engine with an initial Damage Capacity value of 18.
+
+You may add +1 to the result of Leadership tests, even if the Great Gargant's current Damage Capacity value falls below 15.
+====
+
+.Imperial Guard Super Heavy Tank Company
+====
+Baneblades are war engines with an initial Damage Capacity value of 4.
+
+A detachment of four Baneblades has a total initial Damage Capacity value of 16, so you may add +1 to the result of Leadership tests.
+
+If the detachment is reduced to three Baneblades the total is 12, so you cannot add +1 to the result of Leadership tests.
+====
+
+
```

Comment:
Please double-check that the tweaks in this suggestion don't 'undo' anything or introduce new problems.

_Aside: Note that I use the backtick 'code' format for actual data sheet values, and not for totals of multiple data sheet values._

```suggestion
.Leadership test bonus -- Ork Great Gargant
====
The Great Gargant is a War Engine unit that has an initial Damage Capacity value of `18`.
It forms its own detachment of one unit.

You may add +1 to the results of Leadership tests for this detachment, even after the Great Gargant's current Damage Capacity value falls below 15.
====

.Leadership test bonus -- Imperial Guard Super Heavy Tank Company
====
The Baneblade is a War Engine unit that has an initial Damage Capacity value of `4`.
So, a detachment that currently includes four Baneblade units has an initial Damage Capacity value total of 16.

You may add +1 to the results of Leadership tests for this detachment, even after their current Damage Capacity values total less than 15.

But, if any units are destroyed or no longer in play for some reason, then the initial Damage Capacity values will total less than 15.
So then you may no longer add +1 to the results of Leadership tests.
====

```